### PR TITLE
DOCU-1089 and 1088

### PR DESCRIPTION
**Pull request by stepgerner**
Based on https://jira.wandisco.com/browse/DOCU-1088 and https://jira.wandisco.com/browse/DOCU-1089

I had no clue where to put this stuff honestly, and thought about it for a very long time. I think it's best as its own page, but notifications will require its own page too.

The "email notifications" links are just placeholders, and will be updated when I document that next.

### CODE REVIEW

File: docs/configure-smtp.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+id: configure-smtp
+title: Configuring an SMTP server
+sidebar_label: SMTP Server
+---
+
+## SMTP
+
+Configuring a connection to an SMTP server for LiveData Migrator allows the use of [email notifications](./configuration-ui.md), which notify the user via email when certain events occur in LiveData Migrator.
```
Comment (shonaisbister-disco) [inline docs/configure-smtp.md:9]: Line 9 - tiny thing but avoid Latin abbreviations altogether. In this case, swap via for by.

File: docs/configure-smtp.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+### Configure an SMTP server in the UI
+
+Configure an SMTP server in the UI by accessing the SMTP configuration tab and filling in the necessary fields.
+
+1. Click on the LiveData Migrator instance name (e.g. "localhost") on the dashboard to access the overview page.
```
Comment (shonaisbister-disco) [inline docs/configure-smtp.md:15]: Line 16-18 Format any buttons - like Save and SMTP Settings - in bold.

File: docs/configure-smtp.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+id: configure-smtp
+title: Configuring an SMTP server
```
Comment (shonaisbister-disco) [inline docs/configure-smtp.md:3]: Line 3 - we need to make it more human for users than SMTP. They're doing it to enable email notifications, so this heading (and probably sidebar) should reflect that e.g. Set up email notifications or similar.

Then your intro sentence would, for context, start something like 'configure an SMTP server to enable email notifications....' etc.

**Review by shonaisbister-disco — CHANGES_REQUESTED** [review]
Hey, reads pretty well, some minor updates is all. I'm not sure about the placement of the content but think it's good enough for UAT in develop for now! Likely it's fine, but will see how it looks before suggesting anything different.

Good job!

File: docs/configure-smtp.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
+id: configure-smtp
+title: Configure Notifications
+sidebar_label: Notifications
+---
+
+## Email Notifications
```
Comment (benradstone) [inline docs/configure-smtp.md:7]: Delete this heading and change the title above to "Configure Email Notifications".

Also consider changing "Notifications" to "Email notifications".

File: docs/configure-smtp.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+LiveData Migrator supports sending email notifications whenever certain events occur, such as when a migration stops or falls behind file system events.
+
+To use email notifications:
+
+1. [Configure an SMTP server to send the emails](#configure-an-smtp-server)
```
Comment (benradstone) [inline docs/configure-smtp.md:13]: This link looks wrong as you've got "in the UI" or "through the CLI" now.

File: docs/configure-smtp.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+1. [Opt in to the desired notifications](#opt-in-to-email-notifications)
```
Comment (benradstone) [inline docs/configure-smtp.md:14]: Line 14: Not sure where this link is going either.

File: docs/configure-smtp.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+Use [commands in the CLI](./command-reference.md#notification-commands) to:
+
+* [Subscribe email addresses to notifications](./command-reference.md#notification-email-addresses-add).
+* [Unsubscribe email addresses from notifications](./command-reference.md#notification-email-addresses-remove).
+* [Add notification types to subscribe to](./command-reference.md#notification-email-types-add).
+* [Remove notification types from subscription](./command-reference.md#notification-email-types-remove).
+* [View the available notification types for subscription](./command-reference.md#notification-email-types-show).
+
+
+## Configure Notification Properties
```
Comment (benradstone) [inline docs/configure-smtp.md:97]: This section should either be copied or moved into the LDM configuration page.

File: docs/configure-smtp.md (hunk 5 above)
Comment (stepgerner) [reply docs/configure-smtp.md:13]: Very perceptive, thank you!

**Pull request merged**